spk/tt-rss/Makefile

```diff
@@ -1,6 +1,6 @@
 SPK_NAME = tt-rss
-SPK_VERS = 20160102
-SPK_REV = 10
+SPK_VERS = 20170422
```

Comment:
The problem with switching away from timestamp is that the new version will be lower than the existing installs and will not allow downgrades.

If there is a way to tell DSM that the new version is actually an upgrade, I am all for it.
 
The remove/install is hardly an option because you will lose data unless you do backups first.